test/unit_test/mocks/service_clients_mocks.mocks.dart

```diff
@@ -11,25 +11,27 @@ import 'package:grpc/src/client/channel.dart' as _i5;
 import 'package:grpc/src/client/connection.dart' as _i2;
 import 'package:grpc/src/client/method.dart' as _i7;
 import 'package:mockito/mockito.dart' as _i1;
-import 'package:viam_sdk/src/gen/app/data/v1/data.pb.dart' as _i13;
```

Comment:
_Lots_ of changes to this auto-generated mocks tests after adding the new `MockSpec` above and running `make build_mocks`. Some of the changes add a new `MockAppRobotServiceClient` class; most of the changes are due to changed imports and missing methods on other mocks such as `Shutdown`.